generator/src/googleapis/codegen/languages/java/1.31.0/templates/___package___/___api_className___.java.tmpl

```diff
@@ -0,0 +1,248 @@
+{% language java %}{% collapsenewlines %}{% call_template _file_header %}
+
+package {{ api.module.name }};
+
+/**
+ * Service definition for {{ api.className }} ({{ api.version }}).
+ *
+ * <p>
+ * {{ api.description }}
+ * </p>
+ *
+ * <p>
+ * For more information about this service, see the
+ * <a href="{{ api.documentationLink }}" target="_blank">API Documentation</a>
+ * </p>
+ *
+ * <p>
+ * This service uses {@link {{ api.className }}RequestInitializer} to initialize global parameters via its
+ * {@link Builder}.
+ * </p>
+ *
+ * @since 1.3
+ * @author Google, Inc.
+ */
+@SuppressWarnings("javadoc")
+public class {{ api.className }} extends com.google.api.client.googleapis.services.json.AbstractGoogleJsonClient {
+
+  // Note: Leave this static initializer at the top of the file.
+  static {
+    com.google.api.client.util.Preconditions.checkState(
+        com.google.api.client.googleapis.GoogleUtils.MAJOR_VERSION == 1 &&
+        com.google.api.client.googleapis.GoogleUtils.MINOR_VERSION >= 15,
+        "You are currently running with version %s of google-api-client. " +
+        "You need at least version 1.15 of google-api-client to run version " +
+        "{{ features.releaseVersion }} of the {{ api.title }} library.", com.google.api.client.googleapis.GoogleUtils.VERSION);
```

Comment:
Can you update these versions? This is the first time we've added features that require a later version to the generated client in a long time.

Reply:
done